### ISSUE TITLE
Doesn't work on IE

### ISSUE THREAD

**Issue by mathishuettl** [Alhadis/Accordion#8]
Hello,

does this accordion work with the Internet Explorer?
On my current projekt it works on every browser except in Internet Explorer.

Link: http://p408166.mittwaldserver.info/produkte/?taxonomy=productcategory&termID=3

**Comment by Alhadis** [comment]
Hi @mathishuettl,

Yes, it does work on Internet Explorer. You'll need to be much more specific as to what you mean by "not working". What version of IE are you referring to?

**Comment by mathishuettl** [comment]
Hello,

thanks for you quick reply. I use IE 11.
Look at my Link then you will see it. The Accordion titles are not clickable as example.

**Comment by Alhadis** [comment]
This seems to be a layout issue:

~~~css
.product_container{
	width: 50%;
	float: left;
	padding: 15px;
}
~~~

I can't see a clearfix anywhere. Try adding this to your stylesheet:

~~~css
 .accordion-inner::after{
	content: "";
	display: block;
	clear: both;
}
~~~

The accordion is working on IE11: it's just that it hasn't been able to calculate the boundaries of each section's contents correctly.

**Comment by mathishuettl** [comment]
okay i added a clearfix but i have the same problem...

**Comment by Alhadis** [comment]
You added different code to what I recommended...

Does this happen if you delete this line?

~~~diff
 .product_container{
 	width: 50%;
-	float: left;
 	padding: 15px;
 }
~~~

**Comment by mathishuettl** [comment]
It's still not as expected. I made the change you can check it here: 
http://p408166.mittwaldserver.info/produkte/?taxonomy=productcategory&termID=3

**Comment by Alhadis** [comment]
Ah, I see what the problem is. For some reason, the `accordion` class is missing from the outermost container:

<img src="https://user-images.githubusercontent.com/2346707/28014668-e1febcb8-65b0-11e7-800f-434f5ac80cf5.png" width="806" alt="Figure 1" />

Whereas in Chrome, the class is there:

<img src="https://user-images.githubusercontent.com/2346707/28014693-fd5de272-65b0-11e7-837e-d25a2e4bf7a2.png" width="427" alt="Figure 2" />

Make sure the `accordion` class is added for *all* browsers, and this problem should disappear. =)

**Comment by mathishuettl** [comment]
Hello,

thanks for this good tipp, but I display the class for all browsers... when I watch the source of the page in IE there is the accordion class so I think it gets removed from any javascript, but I don't touch it.

**Comment by Alhadis** [comment]
OH! Shit, I know what's going on. It's because [these lines](https://github.com/Alhadis/Accordion/blob/04fa4ba/src/accordion.js#L651-L652) are assuming the browser supports the second argument of `classList.toggle`. Which [IE never supported](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility). Basically, this lets you add/remove a class based on whether the second argument (if provided) evaluates to true or false.

Which I failed to notice because I'd been using a [bundle of polyfills](https://github.com/Alhadis/Fix-IE) for IE which fixes that. Of course, it's my fault that I forgot it mightn't always be available...

I'll address this and follow up with a patch release. In the meantime, you can [use this](https://github.com/Alhadis/DOMTokenList) to fix the lack of full `.toggle` support in IE 11.

**Comment by mathishuettl** [comment]
Thank you!!

**Comment by Alhadis** [comment]
@mathishuettl Sorry for the long wait, I didn't mean for "a few days" to snowball into an entire week.

This has been released and published as [`v2.1.1`](https://github.com/Alhadis/Accordion/releases/tag/v2.1.1). Thanks again for bringing this up; please let me know if you encounter any more difficulties in future. =)
